src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -482,6 +482,15 @@ internal override void ForceComplete(SourceLocation locationOpt, CancellationTok
                         foreach (var typeParameter in this.TypeParameters)
                         {
                             typeParameter.ForceComplete(locationOpt, cancellationToken);
+                            var diagnostics = DiagnosticBag.GetInstance();
```

Comment:
>var diagnostics = DiagnosticBag.GetInstance(); [](start = 28, length = 46)

It feels like all this added code belongs in Force complete for the type parameter. Looks like CompletionPart.TypeParameterConstraints is the relevant part.

Reply:
Since we decided to enable local functions, the last parameter would be false. Will add that now.

---
In reply to: [172417776](https://github.com/dotnet/roslyn/pull/24817#discussion_r172417776) [](ancestors = 172417776)

Reply:
Consolidated into the other comment.

---
In reply to: [177238909](https://github.com/dotnet/roslyn/pull/24817#discussion_r177238909) [](ancestors = 177238909,175998462,172417776)